src/client/pages/Download/components/DownloadPageContent.tsx

```diff
@@ -130,32 +159,49 @@ export const DownloadPageContent = ({
             ))}
           </TextWrapper>
           <GetAppButtons />
+          {isCrossSellEnabled && <CrossSells />}
         </div>
         {isLargeScreen && (
           <ImageSection>
             <AppImage width={IMAGE_WIDTH} />
           </ImageSection>
         )}
-
-        {isCrossSellEnabled && <CrossSells />}
       </ContentContainer>
     </Page>
   )
 }
 
 const CrossSells = () => {
   const { loading, crossSells } = useCrossSells()
+  const textKeys = useTextKeys()
 
   if (loading) return null
 
   return (
-    <div style={{ textAlign: 'center' }}>
+    <>
+      <CrossSellSectionHeader>
+        {textKeys.ONBOARDING_CROSS_SELL_HEADER()}
+      </CrossSellSectionHeader>
       {crossSells?.map((crossSell) => (
-        <div key={crossSell.id}>
-          <p>{crossSell.title}</p>
-          <p>{crossSell.description}</p>
-        </div>
+        <StyledLinkCard
+          key={crossSell.id}
+          href={crossSell.type}
+          orientation="column"
+        >
+          <ImageFrame>
+            <Image src={crossSell.image} />
+          </ImageFrame>
+          <Section>
+            <Header>
+              <Title>{crossSell.title}</Title>
+            </Header>
+            <Description>{crossSell.description}</Description>
+            <Cta>
+              {crossSell.callToAction} <Arrow direction="forward" />
```

Comment:
kinda looks like the arrow is slightly too big to align well with the text - maybe make it a big smaller?

Reply:
Sure, I can tweak the size 👍